conditionals.py

```diff
@@ -33,10 +29,16 @@
    # Write conditional statements that check which is the highest score and prints it.
 
 
+1 = (random.randrange(1, 10))
+2 = (random.randrange(1, 10))
+3 = (random.randrange(1, 10))
 
-
-
-
+if one> two and one> three
+   print ("one wins")
+elif two> one and two> three
+   print ("two wins")
+else 
+   print ("three wins")
```

Comment:
This line of code would also cause an error. Add colons (`:`) at the end of each conditional statement. 

```suggestion
if one> two and one> three:
   print ("one wins")
elif two> one and two> three:
   print ("two wins")
else: 
   print ("three wins")
```

Reply:
completed